lib/hanami/action/params.rb

```diff
@@ -90,6 +90,14 @@ def error_messages(error_set = errors)
         end
       end
 
+      # Returns true if params is valid
```

Comment:
What if params in not invalid? What do you think about restructuring this doc comment to reveal the intention of the method rather than the specific implementation behaviour?